pkg/cmd/drt/scripts/start_all_workloads.sh

```diff
@@ -2,3 +2,4 @@ sudo systemd-run --working-directory=/home/ubuntu --service-type exec --collect
 sudo systemd-run --working-directory=/home/ubuntu --service-type exec --collect --unit cct_tpcc ./tpcc_run.sh
```

Comment:
Let's add uids/gids for all of these, as they all need it now

Reply:
Done.  I also added the schemachange workload.  PTAL.